idf_component.yml

```diff
@@ -0,0 +1,8 @@
+version: "0.1.0"
```

Comment:
Must this file be put under the root directory? There are too many files under root dir now.

Reply:
Yes, this is need to be under the root directory, but version can be WAMR version plus bugfix version, for example now WAMR version is 1.3.2, we can make this to be 1.3.2.x(x is bugfix version is needed).